### PR TITLE
Polished up subsets.md

### PR DESCRIPTION
**Pull request by SuperGamer18**
https://github.com/SuperGamer18/RetroAchievements-docs/blob/updates/docs/guidelines/content/subsets.md
A father simple edit, removing stuff such as unnecessary line breaks and all those spaces in the Approval and Claims table, among other minor stuff.

### CODE REVIEW

File: docs/guidelines/content/subsets.md
Comment (suspect15) [inline docs/guidelines/content/subsets.md]: The actual format of the subset on site is Ninja Gaiden | Shadow Warriors [Subset - Full Game Damageless], so the bracketed portion should be retained.  This update would misalign the presentation with the other examples in the Naming Scheme section.

This particular edit should be undone.

File: docs/guidelines/content/subsets.md
Comment (suspect15) [inline docs/guidelines/content/subsets.md:132]: This section is under DevCompliance authority and not something that can be removed during a clean up edit.  The Notes section must be restored.

**Review by suspect15 — CHANGES_REQUESTED** [review]
2 minor issues, otherwise clean up looks good.

File: docs/guidelines/content/subsets.md
Comment (SuperGamer18) [reply docs/guidelines/content/subsets.md:132]: That section was redundant, as ¹ said essentially the same thing but with better formatting (yes really, compare both of them).

File: docs/guidelines/content/subsets.md
Comment (SuperGamer18) [reply docs/guidelines/content/subsets.md]: Well, I'm not the one who has formatted it into the state previous to editing this so...

**Review by redwizard42 — COMMENTED** [review]
I agree some cleanup is needed, but Developer Compliance will handle it alongside some potential future changes. I'd rather this not be changed at this time.

File: docs/guidelines/content/subsets.md
Comment (redwizard42) [reply docs/guidelines/content/subsets.md:132]: Sometimes redundancy is intentional to ensure the consumer catches the rule. In any case, please do not edit devcoc content.

**Comment by SuperGamer18** [comment]
Well, I guess the best option is to turn this into a draft for now...

File: docs/guidelines/content/subsets.md
Comment (SuperGamer18) [reply docs/guidelines/content/subsets.md:132]: Before @suspect15 said that, I wasn't aware about that, so I'll revert this and turn it into a draft for now.

**Comment by wescopeland** [comment]
@suspect15 @redwizard42 @SuperGamer18 I'm unsure of the status of this PR. Are more revisions pending to this branch?

**Comment by SuperGamer18** [comment]
The reason I'm turning this into a draft instead of outright closing it is mainly due to what @redwizard42 has said to me, particularly the following:
> I agree some cleanup is needed, but Developer Compliance will handle it alongside some potential future changes. I'd rather this not be changed at this time.

Also don't mind suspect15 being requested for review, that was before redwizard was involved (I wasn't expecting him at all). So, for now, it's on a hiatus, aka I'm not sure when should I "reopen" this issue. The only commits you'll see here until I get full approval will be updating my branch, for now...

**Comment by redwizard42** [comment]
If you want to do basic format cleanup and typo fix, that's probably fine as long as it's in-line with other document standards. Any content change should be avoided (also make sure not to incorrectly name the example sunset -- the only fix there is to add Shadow Warriors to the name because that game renamed happened after the example was used).

**Comment by SuperGamer18** [comment]
Done. Didn't even have to add a backslash. Any other changes are welcome :3

**Review by redwizard42 — APPROVED** [review]

**Comment by suspect15** [comment]
More has been added since I last looked including some things that may be at odds with an upcoming DevCompliance updated.  While there may be some room in the doc post DC update for some of the proposals in this PR, I think most of them will be addressed.

My recommendation is to close this PR and wait until the DC update is merged, then if there is room for improvement, open a new PR.

To avoid this sort of issue in the future, it might be a good idea to make a list of docs you intend to work on and we can let you know which ones are unlikely to be updated by teams soon so we don't run into an issue where there was likely wasted effort due to lack of prior coordination.

**Review by suspect15 — APPROVED** [review]
DC will be making a fairly significant update to this doc in the next few days, but the edits by SuperGamer are value added for what the doc currently provides guidance on and can be used as a baseline for the DC updates.  Recommend either merging soon or closing to avoid conflicts with a DC update later this week.

**Comment by wescopeland** [comment]
If this is removed from draft, I can merge.

**Comment by SuperGamer18** [comment]
Done. Feel free to merge. Also not gonna lie I wasn't expecting a direct commit to my fork from someone who isn't me (as in a person making a commit that isn't taken from me merging the changes from the main repo over to my fork)

**Pull request merged**
